### ISSUE TITLE
Phone slows down after a day or two

### ISSUE THREAD

**Issue by DRA7** [lululoid/LMKD-PSI-Activator#4]
So i tried this module and I've noticed my phone slowing down after a day or two when i reboot it gets back to normal.
Is this normal? 

![Screenshot_20241115-113246_DiskInfo](https://github.com/user-attachments/assets/23e478e7-0594-492a-afeb-c890a32b07cc)

This is what diskinfo shows

**Comment by lululoid** [comment]
what phone?

**Comment by DRA7** [comment]
Not s10 plus exynos version

**Comment by DRA7** [comment]
My problem is the battery draining I've been having 3 to 4 hours of screen on time because i have android 14 lineageOS 21 rom on my phone and I'm trying to figure out a way to slow down the drain and boost performance

**Comment by lululoid** [comment]
Try this module. It helped my phone, it's originally for MIUI but maybe just give it a try, [Fog Improvement Module](https://github.com/lululoid/fogimp)

**Comment by DRA7** [comment]
Does this improve the battery?

**Comment by DRA7** [comment]
Okay i will install and see what happens I'll keep you updated tyy

**Comment by DRA7** [comment]
I installed it so far it's pretty smooth i feel the ui is responding nicely I'll keep you updated

**Comment by DRA7** [comment]
I wanted to ask you if you know anything that can help with my battery draining issues

**Comment by lululoid** [comment]
is it freshly installed ROM?

**Comment by DRA7** [comment]
No

**Comment by DRA7** [comment]
I had this rom since September this year, and this is day two of using that module you recommend the phone is still smooth and nice without restarting. I wanted to ask if i use this module would it interfere with yours? [https://github.com/PS2ClassicsVault/Dynamic-System-Tweaks-Magisk-Module](url)

**Comment by lululoid** [comment]
I tried it, it's boot loop on my phone, probably don't use it

**Comment by lululoid** [comment]
> I wanted to ask you if you know anything that can help with my battery draining issues 

I have issue with battery in the but usually only after freshly installed a MIUI ROM, so I execute this command in terminal app `su -c "cmd package bg-dexopt-job"`. It rebuild dalvik deodex or something, you may want to give it a try for your battery issue, the details can be found here if you're interested [https://github.com/KelvinCrag/Optimizer](https://github.com/KelvinCrag/Optimizer)

**Comment by DRA7** [comment]
Hey i wanted to say thank you that module is working fine on my phone it made my phone so much smoother, thanks!

**Comment by lululoid** [comment]
It's been integrated now, if it asks for smoothies just choose yes, yes I recommended it to keep RAM usage stable

**Comment by lululoid** [comment]
Yeah, it's a module, not permanent, this module only change a few props and enable existing feature

**Comment by DRA7** [comment]
Hey i don't wanna open another issue but i was wondering if i could use https://github.com/VR-25/zram-swap-manager along with your modules in zram swap manager i have these settings:

disksize=4G
vm="page-cluster=0 swappiness=100 watermark_boost_factor=0 watermark_scale_factor=125"
